CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- /!\ Folder architecture for alerting-rules changed to fit with areas and teams.
```

Comment:
I like it. What I think would make sense when you move it out of wip is that you explain what we are trying to achieve and where we want to go and ask for the other team's reviews especially about the ownership. WDYT?